src/main/java/org/logstash/beats/BeatsHandler.java

```diff
@@ -139,21 +142,13 @@ private String format(String message) {
         InetSocketAddress local = (InetSocketAddress) context.channel().localAddress();
         InetSocketAddress remote = (InetSocketAddress) context.channel().remoteAddress();
 
-        String localhost;
-        if(local != null) {
-            localhost = local.getAddress().getHostAddress() + ":" + local.getPort();
-        } else{
-            localhost = "undefined";
-        }
-
-        String remotehost;
-        if(remote != null) {
-            remotehost = remote.getAddress().getHostAddress() + ":" + remote.getPort();
-        } else{
-            remotehost = "undefined";
-        }
-
-        return "[local: " + localhost + ", remote: " + remotehost + "] " + message;
+        final String localhost = Objects.isNull(local)
```

Comment:
again, seem unrelated changes?

Reply:
I was touching while formatting the file but since they non-related reverted back to origin state, this file change no longer included.